rts/lra/lra-test/pom.xml

```diff
@@ -35,14 +35,14 @@
              -Xrunjdwp:transport=dt_socket,address=${coordinator.debug.port},server=y,suspend=y
              -->
         <coordinator.debug.port>8787</coordinator.debug.port>
-        <coordinator.debug.jdwp>-Xrunjdwp:transport=dt_socket,address=${coordinator.debug.port},server=y,suspend=y</coordinator.debug.jdwp>
+        <coordinator.debug.jdwp>-Xrunjdwp:transport=dt_socket,address=${coordinator.debug.port},server=y,suspend=n</coordinator.debug.jdwp>
         <coordinator.debug.params></coordinator.debug.params>
 
         <!-- config for the test JAX-RS resource, for exampe to enable debugging use
```

Comment:
do we need here this comment when the `swarm.debug.jdwp` attribute covers this?

Reply:
What do you mean, do you want me to add an explanation of how the debugging config works?

Reply:
sure. I'm sorry for not being clear. I just meant that if there is declared `coordinator.debug.jdwp` with debug parameters the comment could be removed completely. The information is doubled.